core/domain/subtopic_page_domain.py

```diff
@@ -89,45 +103,54 @@ def __init__(
         self.recorded_voiceovers = recorded_voiceovers
         self.written_translations = written_translations
 
-    def validate(self):
+    def validate(self) -> None:
         """Validates the SubtopicPageContentsObject, verifying that all
         fields are of the correct type.
         """
-        self.subtitled_html.validate()
+        self.subtitled_html.validate( # type: ignore[no-untyped-call]
+        )
         content_ids = set([self.subtitled_html.content_id])
-        self.recorded_voiceovers.validate(content_ids)
-        self.written_translations.validate(content_ids)
+        self.recorded_voiceovers.validate( # type: ignore[no-untyped-call]
+        content_ids)
+        self.written_translations.validate( # type: ignore[no-untyped-call]
+        content_ids)
 
     @classmethod
-    def create_default_subtopic_page_contents(cls):
+    def create_default_subtopic_page_contents(cls) -> 'SubtopicPageContents':
```

Comment:
```suggestion
    def create_default_subtopic_page_contents(cls) -> SubtopicPageContents'
```

would this work (not sure)?